src/style.css

```diff
@@ -0,0 +1,55 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+.star-container {
+  display: flex;
+  flex-direction: column;
+}
+
+.stars--0,
+.stars--1,
+.stars--2,
+.stars--3,
+.stars--4,
+.stars--5 {
+  display: flex;
+}
```

Comment:
give common class for every line not to duplicate unique one

Reply:
Added common 'stars' class